core/include/crypto/crypto.h

```diff
@@ -172,6 +172,11 @@ struct x25519_keypair {
 	uint8_t *pub;	/* Public value */
 };
 
+struct ed25519_keypair {
+	uint8_t *priv;	/* Private value */
+	uint8_t *pub;	/* Public value */
```

Comment:
Comments not needed, the names are obvious

Reply:
fixed